### ISSUE TITLE
Implement medinfo/common/ProcessManager.py 

### ISSUE THREAD

**Issue by jonc101** [HealthRex/CDSS#143]
Helper module to facilitate some recurrent process management tasks.
For example, support script currently to watch a process and wait until it finishes before spawning another process/command.
Many existing scripts that generate commands (with argv options) and then execute them serially. Develop option to do that while tracking separate log outputs, and another options to spawn a max number of child processes in parallel.

**Comment by jonc101** [comment]
No longer needed
